### PR TITLE
[mattupham/staking validator table tooltips

### PR DESCRIPTION
**Pull request by mattupham**
## What is the purpose of the change

- add tooltip for high APR
- add tooltip for high voting power
- format values for display

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/867866z1y)

## Testing and Verifying

Screenshot 2023-08-03 at 6 28 12 PM
Screenshot 2023-08-03 at 6 28 15 PM
Screenshot 2023-08-03 at 6 30 33 PM
Screenshot 2023-08-03 at 6 30 37 PM
Screenshot 2023-08-03 at 6 58 00 PM

### CODE REVIEW

File: packages/web/modals/validator-squad.tsx [hunk 1]
```diff
@@ -21,37 +21,34 @@ import { ExternalLinkIcon, Icon } from "~/components/assets";
...
 import { ModalBase, ModalBaseProps } from "~/modals/base";
 import { useStore } from "~/stores";
+import { theme } from "~/tailwind.config";
 import { normalizeUrl, truncateString } from "~/utils/string";
 
-interface ExtendedModalBaseProps extends ModalBaseProps {
+interface ValidatorSquadModalProps extends ModalBaseProps {
   usersValidatorsMap: Map<string, Staking.Delegation>;
   validators: Staking.Validator[];
 }
```
Comment (jonator) [inline packages/web/modals/validator-squad.tsx:34]: Nit: It's nice how this extends `ModalBaseProps` as an interface, but conventionally we inline the types in the component signature. 

It's nice to have the types of the props right next to the props so we don't have to scroll up and down constantly.

**Review by jonator — APPROVED** [review]
LGTM

File: packages/web/modals/validator-squad.tsx [hunk 2]
```diff
@@ -196,21 +194,90 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
             {
               accessorKey: "myStake",
               header: () => t("stake.validatorSquad.column.myStake"),
+              cell: observer((props: CellContext<Validator, Validator>) => {
```
Comment (MaxMillington) [inline packages/web/modals/validator-squad.tsx:197]: is the `observer` needed in the `cell`?

File: packages/web/modals/validator-squad.tsx [hunk 3]
```diff
@@ -196,21 +194,90 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
...
+                  .hideDenom(true)
+                  .toString();
+
+                return <>{formattedMyStake}</>;
+              }),
             },
             {
               accessorKey: "votingPower",
               header: () => t("stake.validatorSquad.column.votingPower"),
+              cell: observer((props: CellContext<Validator, Validator>) => {
```
Comment (MaxMillington) [inline packages/web/modals/validator-squad.tsx:214]: same questions ^^

File: packages/web/modals/validator-squad.tsx [hunk 4]
```diff
@@ -196,21 +194,90 @@ const ValidatorSquadContent: FunctionComponent<ValidatorSquadContentProps> =
...
+                  .maxDecimals(2)
+                  .toString();
+
+                return <>{formattedVotingPower}</>;
+              }),
             },
-            {
-              accessorKey: "commissions",
+            columnHelper.accessor((row) => row, {
+              cell: observer((props: CellContext<Validator, Validator>) => {
```
Comment (MaxMillington) [inline packages/web/modals/validator-squad.tsx:226]: same question ^

**Review by MaxMillington — APPROVED** [review]
One question about the observer, otherwise looks good

**Review by MaxMillington — APPROVED** [review]

**Pull request merged**